meta.yml

```diff
@@ -33,36 +33,52 @@ authors:
 - name: Thomas Sibut-Pinote
   initial: true
 
-maintainers:
-- name: Assia Mahboubi
-
-
 opam-file-maintainer: assia.mahboubi@inria.fr
 
-opam-file-version: released
+opam-file-version: dev
 
 license:
   fullname: CeCILL-C
   identifier: CECILL-C
   file: Licence_CeCILL-C_V1-en.txt
 
 supported_coq_versions:
-  text: 8.11 or later (use releases for other Coq versions)
-  opam: '{>= "8.11"}'
-
+  text: 8.11
+  opam: '{>= "8.11" & < "8.12~"}'
 
 tested_coq_opam_versions:
-- version: '8.11'
+- version: '1.10.0-coq-8.11'
+  repo: 'mathcomp/mathcomp'
 
 dependencies:
+- opam:
```

Comment:
Should I provide explicitly the description of all dependencies? Is it (only) for a better looking README?